ioctl/client.go

```diff
@@ -147,8 +151,34 @@ func (c *client) SelectTranslation(trls map[config.Language]string) (string, con
 }
 
 func (c *client) ReadSecret() (string, error) {
-	// TODO: delete util.ReadSecretFromStdin, and move code to here
-	return util.ReadSecretFromStdin()
+	signalListener := make(chan os.Signal, 1)
```

Comment:
revert this change. you can do it in a different PR or even leave as it is until we migrate to the new command

Reply:
ok